--- headers/openvr.h	Tue Oct 16 02:14:19 2018
+++ headers/openvr.h	Sun Oct 28 11:07:09 2018
@@ -1535,10 +1535,14 @@
 // figure out how to import from the VR API dll
 #if defined(_WIN32)
 
+#ifdef VR_BUILD_SHARED 
 #ifdef VR_API_EXPORT
 #define VR_INTERFACE extern "C" __declspec( dllexport )
 #else
 #define VR_INTERFACE extern "C" __declspec( dllimport )
+#endif
+#else
+#define VR_INTERFACE extern "C"
 #endif
 
 #elif defined(__GNUC__) || defined(COMPILER_GCC) || defined(__APPLE__)
